Documentation/edge-ai.md

```diff
@@ -1,13 +1,43 @@
+# What is Edge-AI?
+Currently, the heavy computing capacity required to run deep learning models necessitates that the majority of AI processes be carried out in the cloud.
 
-This is WIP page.
+However, running AI in the cloud has its disadvantages, including the fact that it requires an internet connection, and that performance can be impacted by bandwidth and latency limitations.
 
-# What is Edge-AI?
+Edge AI, also known as ‘on-device AI’, is a distributed computing paradigm that allows for AI algorithms to be run locally on a device, using the data produced by that device.
+
+Running AI at the ‘edge’ of the local network removes the requirement for the device to be connected to the internet or centralized servers like the cloud.
```

Comment:
Q. What does 'the edge of the local network' mean?

Reply:
It may be a single edge device, but multiple edge devices connected locally can run AI tasks.
If this causes confusion from readers, I will change to `at the edge device`